source/adaptive_load/input_variable_setter_impl.cc

```diff
@@ -0,0 +1,38 @@
+#include "adaptive_load/input_variable_setter_impl.h"
+
+#include "envoy/registry/registry.h"
+
+#include "external/envoy/source/common/protobuf/protobuf.h"
+
+namespace Nighthawk {
+
+RequestsPerSecondInputVariableSetter::RequestsPerSecondInputVariableSetter(
+    const nighthawk::adaptive_load::RequestsPerSecondInputVariableSetterConfig&) {}
```

Comment:
just currious, if RequestsPerSecondInputVariableSetterConfig is not used, do you still need this constructor?

Reply:
Good point, see #438.